pkg/decorate/compat/names.go

```diff
@@ -36,8 +19,8 @@ func (n Names) Name() string {
 func (Names) Decorate(mfs []*dto.MetricFamily) {
 	for _, mf := range mfs {
 		n := strings.ToLower(mf.GetName())
-		if newName, ok := nameConversions[n]; ok {
-			mf.Name = &newName
+		if strings.HasPrefix(n, "node_") {
```

Comment:
are we sure there weren't other metrics we didn't want name converted  before  that will be now?

Reply:
e.g. I see "node_cpu_seconds_total" as a metric, that wasn't name converted before afaict

Reply:
I believe it should have been all our `node_` metrics

Reply:
nevermind there are other problems with doing a prefix replace

Reply:
going back to the old map LUT